pkgs/applications/audio/parlatype/default.nix

```
@@ -0,0 +1,62 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, meson, gtk3, dbus, gst_all_1, ninja, gettext, appstream-glib, python3, desktop-file-utils, glib, gobject-introspection, gsettings-desktop-schemas, isocodes, itstool, libxml2, wrapGAppsHook, hicolor-icon-theme }:
+
+stdenv.mkDerivation rec {
+  pname = "parlatype";
+  version = "3.1";
+
+  src = fetchFromGitHub {
+    owner  = "gkarsay";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "1a4xlsbszb50vnz1g7kf7hl7aywp7s7xaravkcx13csn0a7l3x45";
```

No need for such a pretty-printing.

```suggestion
    owner = "gkarsay";
    repo = pname;
    rev = "v${version}";
    sha256 = "1a4xlsbszb50vnz1g7kf7hl7aywp7s7xaravkcx13csn0a7l3x45";
```